src/platform/plugins/shared/expressions/common/execution/execution.ts

```diff
@@ -125,7 +125,7 @@ function throttle<T>(timeout: number) {
 
       const emit = () => {
         if (hasValue) {
-          subscriber.next(latest);
+          subscriber.next(latest!);
```

Comment:
Since this surfaced as a type error with the new upgrade could be interesting to fix it correctly without an non-null assertion.
To avoid that you can:
- check the existence of `latest` together with the `hasValue` 
- probably better rewrite a bit the function to something like

```
let value: {latest: T, hasValue: true} | {latest: undefined, hasValue: false} = {latest: undefined, hasVaue:false};
// and then use this instead of two separate variables to have better working types
```

Reply:
++ Do you want me to create an issue to address this?